outputs/nexusiq.go

```diff
@@ -0,0 +1,181 @@
+package outputs
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/aquasecurity/postee/v2/formatting"
+	"github.com/aquasecurity/postee/v2/layout"
+)
+
+var notAllowed = regexp.MustCompile(`[\.:\/]`)
```

Comment:
How is this determined?

Reply:
based on https://regex101.com/r/eK9lPd/3 and testing with actual nexus iq api